double-conversion/bignum.cc

```diff
@@ -147,7 +147,7 @@ void Bignum::AssignHexString(Vector<const char> value) {
   }
   if (tmp > 0) {
     DOUBLE_CONVERSION_ASSERT(tmp <= kBigitMask);
-    RawBigit(used_bigits_++) = (tmp & kBigitMask);
+    RawBigit(used_bigits_++) = (Bignum::Chunk)(tmp & kBigitMask);
```

Comment:
Please change to C++ `static_cast<Bignum::Chunk>` casts. Here and for the others.